nodejs-server/src/echoServer.ts

```diff
@@ -0,0 +1,316 @@
+import * as grpc from 'grpc';
+import * as Long from 'long';
+
+import * as echoTypes from '../pbjs-genfiles/echo.js';
+
+import showcaseV1Beta1 = echoTypes.google.showcase.v1beta1;
+import google = echoTypes.google;
+import longrunning = google.longrunning;
+import {OperationsServer} from './operationsServer.js';
+
+/**
+ * Implements Echo server based on echo.proto.
+ */
+export class EchoServer {
+  private requestCount: number;
+  private paginationOutput: {[index: string]: string[]};
```

Comment:
For both of these - why not just use a Map?

Reply:
I always forget about it! Fixed.